cockroach/plan.sh

```diff
@@ -1,12 +1,12 @@
 pkg_name=cockroach
 pkg_origin=core
-pkg_version=2.0.1
+pkg_version=2.0.6
 pkg_description="CockroachDB is a cloud-native SQL database for building global, scalable cloud services that survive disasters."
 pkg_upstream_url=https://github.com/cockroachdb/cockroach
 pkg_license=('Apache-2.0')
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_source=https://binaries.cockroachdb.com/cockroach-v${pkg_version}.src.tgz
-pkg_shasum=4f929461ad50d3fd471886856aa399f41a405a664e886cf9ab71d2879162955e
+pkg_shasum=fd1d966542565cbeca6c390db6b7afd98bc239e6e2a4602b3edff973be472097
 pkg_dirname="${pkg_name}-v${pkg_version}"
```

Comment:
Should we add `pkg_svc_user`?  If I'm understanding the ports it uses correctly,  it doesn't need to bind to a privileged port.

Reply:
Ports 8080, 26257 don't need `root`.

Reply:
Long week,  default user is hab not root. My question has no meaning.